From 6cf9765d7586af4321a5cec681bc2875e029068a Mon Sep 17 00:00:00 2001
From: Yann Gautier <yann.gautier@foss.st.com>
Date: Wed, 12 Mar 2025 19:11:48 +0100
Subject: [PATCH 849/954] docs: rename all 'rss' files to 'rse'

Upstream-Status: Inappropriate [ST downstream https://github.com/STMicroelectronics/arm-trusted-firmware.git 822842a17794cb312f3bca47b8c823e02f42926c docs: rename all 'rss' files to 'rse']
Signed-off-by: Tamas Ban <tamas.ban@arm.com>
Change-Id: I93877ebeca1db6ee27dcb5446cc1f1a1e4e56775
(cherry picked from commit a5a5947a283331f5d99ef4be80393c01826d10dd)
Reviewed-on: https://gerrit.st.com/c/mpu/oe/st/tf-a/+/443841
Domain-Review: Yann GAUTIER <yann.gautier@foss.st.com>
Reviewed-by: Maxime MERE <maxime.mere@foss.st.com>
Tested-by: Yann GAUTIER <yann.gautier@foss.st.com>
ACI: CIBUILD <MDG-smet-aci-builds@list.st.com>
ACI: CITOOLS <MDG-smet-aci-reviews@list.st.com>
Reviewed-by: Yann GAUTIER <yann.gautier@foss.st.com>
---
 docs/design_documents/{rss.rst => rse.rst}                        | 0
 .../{rss_attestation_flow.puml => rse_attestation_flow.puml}      | 0
 .../{rss_measured_boot_flow.puml => rse_measured_boot_flow.puml}  | 0
 .../diagrams/plantuml/{tfa_rss_dfd.puml => tfa_rse_dfd.puml}      | 0
 .../{rss_attestation_flow.svg => rse_attestation_flow.svg}        | 0
 .../{rss_measured_boot_flow.svg => rse_measured_boot_flow.svg}    | 0
 ...eat_model_rss_interface.rst => threat_model_rse_interface.rst} | 0
 7 files changed, 0 insertions(+), 0 deletions(-)
 rename docs/design_documents/{rss.rst => rse.rst} (100%)
 rename docs/resources/diagrams/plantuml/{rss_attestation_flow.puml => rse_attestation_flow.puml} (100%)
 rename docs/resources/diagrams/plantuml/{rss_measured_boot_flow.puml => rse_measured_boot_flow.puml} (100%)
 rename docs/resources/diagrams/plantuml/{tfa_rss_dfd.puml => tfa_rse_dfd.puml} (100%)
 rename docs/resources/diagrams/{rss_attestation_flow.svg => rse_attestation_flow.svg} (100%)
 rename docs/resources/diagrams/{rss_measured_boot_flow.svg => rse_measured_boot_flow.svg} (100%)
 rename docs/threat_model/{threat_model_rss_interface.rst => threat_model_rse_interface.rst} (100%)

diff --git a/docs/design_documents/rss.rst b/docs/design_documents/rse.rst
similarity index 100%
rename from docs/design_documents/rss.rst
rename to docs/design_documents/rse.rst
diff --git a/docs/resources/diagrams/plantuml/rss_attestation_flow.puml b/docs/resources/diagrams/plantuml/rse_attestation_flow.puml
similarity index 100%
rename from docs/resources/diagrams/plantuml/rss_attestation_flow.puml
rename to docs/resources/diagrams/plantuml/rse_attestation_flow.puml
diff --git a/docs/resources/diagrams/plantuml/rss_measured_boot_flow.puml b/docs/resources/diagrams/plantuml/rse_measured_boot_flow.puml
similarity index 100%
rename from docs/resources/diagrams/plantuml/rss_measured_boot_flow.puml
rename to docs/resources/diagrams/plantuml/rse_measured_boot_flow.puml
diff --git a/docs/resources/diagrams/plantuml/tfa_rss_dfd.puml b/docs/resources/diagrams/plantuml/tfa_rse_dfd.puml
similarity index 100%
rename from docs/resources/diagrams/plantuml/tfa_rss_dfd.puml
rename to docs/resources/diagrams/plantuml/tfa_rse_dfd.puml
diff --git a/docs/resources/diagrams/rss_attestation_flow.svg b/docs/resources/diagrams/rse_attestation_flow.svg
similarity index 100%
rename from docs/resources/diagrams/rss_attestation_flow.svg
rename to docs/resources/diagrams/rse_attestation_flow.svg
diff --git a/docs/resources/diagrams/rss_measured_boot_flow.svg b/docs/resources/diagrams/rse_measured_boot_flow.svg
similarity index 100%
rename from docs/resources/diagrams/rss_measured_boot_flow.svg
rename to docs/resources/diagrams/rse_measured_boot_flow.svg
diff --git a/docs/threat_model/threat_model_rss_interface.rst b/docs/threat_model/threat_model_rse_interface.rst
similarity index 100%
rename from docs/threat_model/threat_model_rss_interface.rst
rename to docs/threat_model/threat_model_rse_interface.rst
-- 
2.53.0

